Assets/Mirror/Editor/Weaver/Processors/MessageClassProcessor.cs

```diff
@@ -6,6 +6,19 @@ namespace Mirror.Weaver
 {
     static class MessageClassProcessor
     {
+
+        static OpCode[] emptyBodyTemplate = { OpCodes.Nop, OpCodes.Ret };
```

Comment:
```suggestion
        static readonly OpCode[] emptyBodyTemplate = { OpCodes.Nop, OpCodes.Ret };
```

Reply:
readonly adds a tiny overhead while not granting any advantage here, as the array can be manipulated nonetheless, so i went with not adding it, but it's reasonable if it provides some visual indication for others

Reply:
Do you have any benchmark or article describing this overhead? First time I hear about it

Reply:
As noted, it's tiny, but having read that a while ago (https://codeblog.jonskeet.uk/2014/07/16/micro-optimization-the-surprising-inefficiency-of-readonly-fields/) i made it a habit of not using it as i rarely even use it for it's actual functionality (using get only properties most of the time)

Reply:
That's talking about not readonly structs used in readonly fields, this isn't even a struct.

Reply:
You're right, looks like i misread it back then and kept the wrong idea